compiler/bindings/python/test/tools/CMakeLists.txt

```diff
@@ -15,6 +15,15 @@ iree_py_test(
 )
 endif()  # IREE_BUILD_BUNDLED_LLVM
 
+if(IREE_INPUT_TORCH)
+  iree_py_test(
+    NAME
+      import_onnx_test
```

Comment:
Future cleanup once this settles... we could add a `IREE_INPUT_ONNX` that depends on `IREE_INPUT_TORCH` and use that here instead.

I also wonder if we will have multiple paths from ONNX to IREE/Linalg. Guess we'll cross that bridge later. Just getting any path working at first matters more.

Reply:
Agreed. Once this lands and some of the upstream work settles, we need to add a slightly modified input lowering pipeline for ONNX anyway. Will get there when we get there.